.github/workflows/ci.yml

```diff
@@ -38,21 +27,7 @@ jobs:
       - uses: actions/checkout@v5
 
       - name: Setup toolchain
-        uses: dtolnay/rust-toolchain@master
-        with:
-          toolchain: nightly
-          components: rustfmt
-
-      - name: Cache cargo
-        uses: actions/cache@v4
-        with:
-          path: |
-            ~/.cargo/registry
-            ~/.cargo/git
-            target
-          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}-nightly
-          restore-keys: |
-            ${{ runner.os }}-cargo-
+        uses: actions-rust-lang/setup-rust-toolchain@v1
```

Comment:
This action will configure Rust cache automatically: https://github.com/actions-rust-lang/setup-rust-toolchain?tab=readme-ov-file#inputs